sessions/recap-project-5/example-implementation/components/ArtPiecesPreview.js

```diff
@@ -0,0 +1,65 @@
+import Image from "next/image";
+import Link from "next/link";
+import styled from "styled-components";
+import FavoriteButton from "./FavoriteButton";
+
+const ImageContainer = styled.figure`
+  position: relative;
+  width: 100%;
+  height: 100%;
+  margin: 0;
+`;
+
+const ImageCaption = styled.figcaption`
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  width: 100%;
+  padding: 0.3rem;
+  background: #000;
+  color: #fff;
+  bottom: 1rem;
+
+  & span:last-child {
+    color: grey;
+  }
+`;
+
+const StyledImage = styled(Image)`
+  object-fit: cover;
+`;
+
+const ImageWrapper = styled.div`
+  position: relative;
+`;
+
+function ArtPiecesPreview({ piece, handleClickToggleFavorite, isFavorite }) {
```

Comment:
I'd prefer a naming indicating singular. This component renders just one piece.

```suggestion
function ArtPiecePreview({ piece, handleClickToggleFavorite, isFavorite }) {
```